--- lbxproxy-1.0.2/Makefile.am.orig	2009-11-03 07:23:21.000000000 +0100
+++ lbxproxy-1.0.2/Makefile.am	2009-11-06 06:52:00.524440813 +0100
@@ -23,8 +23,9 @@
 
 AM_CFLAGS = $(CWARNFLAGS)
 
+configdir = $(libdir)/X11/lbxproxy
 AM_CPPFLAGS = $(LBXPROXY_CFLAGS) $(OS_CFLAGS) -I$(top_srcdir)/include	\
-	-DDEF_ATOMS_FILE=\"$(libdir)/X11/lbxproxy/AtomControl\" \
+	-DDEF_ATOMS_FILE=\"$(configdir)/AtomControl\" \
 	-D_BSD_SOURCE -DLBXPROXY_t -DTRANS_SERVER
 
 lbxproxy_LDADD = $(LBXPROXY_LIBS)
@@ -83,7 +84,6 @@
 appman_PRE = \
         lbxproxy.man
 
-configdir = $(libdir)/X11/lbxproxy
 dist_config_DATA = config/AtomControl
 
 EXTRA_DIST = \
